glyphs-reader/Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-plist_derive = { path = "plist_derive" }
+ascii_plist_derive = { path = "ascii_plist_derive" }
```

Comment:
this is also commonly referred to as OpenStep plist format. Given that our own parser (as well as Glyphs.app's) happily accept UTF-8 literal strings in addition to plain ASCII, I would use that name if we can

Reply:
This is the original PLIST format used by NeXTSTEP (and then OpenStep) but I think it's fair to refer to it as 'ASCII', since this format is commonly referred to as "ascii-style" plists, to differentiate them from the XML format that apple adopted later, but I'll update the README to mention NeXTSTEP/OpenStep..